### PR TITLE
Remove asc link.

### PR DESCRIPTION
**Pull request by ismailsunni**
Remove download ASC file link.

### CODE REVIEW

**Comment by ismailsunni** [comment]
@Gustry thanks, I forgot to remove them. Removed now.

File: geonode/settings.py [hunk 1]
```diff
@@ -881,7 +881,6 @@
     'Tiles',
     'GML',
     'GZIP',
-    'ASCII',
```
Comment (Gustry) [inline geonode/settings.py]: Is-it OK to remove from settings from geonode here ?

Comment (ismailsunni) [inline geonode/settings.py]: It was my code, I think it should be fine, and it was not specific to qgis server backend also

Comment (Gustry) [inline geonode/settings.py]: ok.

**Pull request merged**